spec/yaml/swiftnav/sbp/telemetry.yaml

```diff
@@ -0,0 +1,129 @@
+package: swiftnav.sbp.telemetry
+description: Telemetry messages reported by Starling engine.
+  The messages include various byproducts of state estimation and other logic across Starling
+  and are aimed at efficient issue diagnostics.
+stable: False
+public: True
+include:
+  - gnss.yaml
+definitions:
+
+  - TelemetrySV:
+      short_desc: Array of per-signal telemetry entries
+      fields:
+        - az:
+            type: u8
+            units: deg * 2g
```

Comment:
Typo? Should this be `deg * 2`?

Reply:
yes, it is indeed `deg * 2`: already corrected!